README.md

```diff
@@ -32,6 +32,8 @@ const defaults = {
   username: null,
   password: null,
   deltaStreamBehaviour: 'none',
+  pingEnable: false,
```

Comment:
I think we could do with just one parameter, `pingInterval`. If not set or set to zero no ping is used.